### PR TITLE
[BLOG] add cxx numba interoperability

### PR DESCRIPTION
**Pull request by noatamir**
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

## Links to other Quansight blogs:

no labs links found
no llc links found

### CODE REVIEW

**Comment by noatamir** [comment]
@gabalafou another blog

File: apps/labs/posts/2021/10/cxx-numba-interoperability.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
+---
+title: 'An efficient method of calling C++ functions from numba using clang++/ctypes/rbc'
+published: October 19, 2021
+author: pearu-peterson
+description: 'An exploration of a method to call C++ libarary function from Numba compiled functions, i.e. python function that are decorated with "numba.jit(nopython=True)"'
+category: [Beyond PyData, PyData Ecosystem]
+featuredImage:
+  src: /posts/2010/10/cxx-numba-interoperability/blog_feature_var2.svg
+  alt: 'An illustration of brown and a white coming towards each other to pass a business card with the logo of Quansight Labs'
```
Comment (MarsBarLee) [inline apps/labs/posts/2021/10/cxx-numba-interoperability.md:9]:
```suggestion
  alt: 'An illustration of a brown and a white hand coming towards each other to pass a business card with the logo of Quansight Labs'
```

**Comment by noatamir** [comment]
@gabalafou another blog with unclear failed build.

**Comment by bskinn** [comment]
This one is an HTTP 401 UNAUTHORIZED in the guts of Vercel, raising an "ApolloError".

**Review by gabalafou — APPROVED** [review]
Todo after merging and deploying:

- [ ] Check that the file links (cxx2py.py, etc.) work

**Comment by trallard** [comment]
This is now working - merging now 🚀

**Comment by gabalafou** [comment]
One of the links in the blog post is broken: https://raw.githubusercontent.com/Quansight/Quansight-website/public/posts/cxx-numba-interoperability/cxx2py_libfoo.cpp

**Comment by gabalafou** [comment]
Fix for links in https://github.com/Quansight/Quansight-website/pull/478

**Pull request merged**